src/olympia/addons/tests/test_serializers.py

```diff
@@ -354,10 +353,17 @@ def test_is_disabled(self):
         assert result['is_disabled'] is True
 
     def test_is_source_public(self):
-        self.addon = addon_factory(view_source=True)
+        self.request.version = 'v3'
+        self.addon = addon_factory()
         result = self.serialize()
```

Comment:
You shouldn't need to set `self.request.version` to activate the shim - that's what the `override_settings` later on is for.